### DISCUSSION TITLE
Is Authentication for GraphQL Datasources supported?

### DISCUSSION THREAD

**Discussion by Sprinting** [ToolJet/ToolJet#3791] [Q&A]
HI, 
I was having a look at tooljet, and while REST Apis do seem to have authentication options available, there is none for GraphQL data sources. Is it simply not supported yet, or am I looking at the wrong place?

**Comment by rafferty-m** [comment]
Bumping this. I have the same question - is this not supported in any way? Are there plans to support it? Are there workarounds?

**Comment by withshubh** [comment]
Hey @Sprinting & @rafferty-m 👋  
Thanks for creating this discussion. We have added the support for Authentication to the roadmap.
We will update here once we release it.
